src/app/dashboard/dashboard.component.html

```diff
@@ -1,28 +1,20 @@
 <div class="row small-up-2 medium-up-2 large-up-2 xlarge-up-3">
-
-  <section class="column">
-  <!-- ng-repeat="projectsbuild in projectsbuilds" -->
-    <div class="project project-success" ng-class="{
-      'project-success': projectsbuild.lastBuild.worker.status == 'Succeeded',
-      'project-failed': projectsbuild.lastBuild.worker.status == 'Failed',
-      'project-active': projectsbuild.lastBuild.worker.status == 'Active'
-    }">
-      <a href="">
-
+  <section class="column" *ngFor="let project of projects">
+    <div class="project">
+      <a routerLink="/project/{{project.id}}">
         <div class="status">
-          <div class="success"><i class="icon ion-md-checkmark-circle status-success"></i></div>
+          <i [ngClass]="calculateStatusClasses(project.lastBuild)"></i>
```

Comment:
@flynnduism this calls the `calculateStatusClasses` function in the dashboard component to figure out which classes to apply to the icon element.